resources/lib/sosac.py

```diff
@@ -20,14 +20,14 @@
 # *
 # */
 
-import re,os,urllib,urllib2,cookielib
+import re,os,urllib,urllib2,cookielib,json,xbmcaddon
```

Comment:
odstranim to ak to tam nikde nepouzivam. json som predtym pouzival ale uz nepouzivam....